aeron-driver/src/main/c/agent/aeron_driver_agent.c

```diff
@@ -1323,22 +1288,11 @@ static const char *dissect_cmd_in(int64_t cmd_id, const void *message, size_t le
             break;
         }
 
-        case AERON_COMMAND_CLIENT_CLOSE:
```

Comment:
Should this be removed?

Reply:
The `AERON_COMMAND_CLIENT_CLOSE` is merged together with `AERON_COMMAND_CLIENT_KEEPALIVE` as they have the same output structure.